lib/routes/veeam/get.js

```diff
@@ -25,21 +26,44 @@ function getVeeamFile(request, response, bucketMd, log) {
         if (err) {
             return responseXMLBody(errors.InternalError, null, response, log);
         }
-        const fileToBuild = getFileToBuild(request, data._capabilities?.VeeamSOSApi);
-        if (fileToBuild.error) {
-            return responseXMLBody(fileToBuild.error, null, response, log);
-        }
-        let modified = new Date().toISOString();
-        // Extract the last modified date, but do not include it when computing
-        // the file's ETag (md5)
-        modified = fileToBuild.value.LastModified;
-        delete fileToBuild.value.LastModified;
+        const finalizeRequest = bucketMetrics => {
+            const fileToBuild = getFileToBuild(request, data._capabilities?.VeeamSOSApi);
+            if (fileToBuild.error) {
+                return responseXMLBody(fileToBuild.error, null, response, log);
+            }
+
+            // Extract the last modified date, but do not include it when computing
+            // the file's ETag (md5)
+            const modified = fileToBuild.value.LastModified;
+            delete fileToBuild.value.LastModified;
 
-        const builder = new xml2js.Builder({
-            headless: true,
-        });
-        return respondWithData(request, response, log, data,
-            buildHeadXML(builder.buildObject(fileToBuild.value)), modified);
+            // The SOSAPI metrics are dynamically computed using the SUR backend.
+            if (bucketMetrics && !fileToBuild.value.CapacityInfo?.Used) {
+                fileToBuild.value.CapacityInfo.Used = Number(bucketMetrics.bytesTotal);
+                fileToBuild.value.CapacityInfo.Available =
+                    Number(fileToBuild.value.CapacityInfo.Capacity) - Number(bucketMetrics.bytesTotal);
+                // TODO CLDSRV-633 when SUR backend supports realtime metrics: enable this, it will
+                // report the real last modified date of the bucket, instead of the current date,
```

Comment:
we don't want to report the `last modified date of the bucket`, as it would indicate stale data if the bucket has not been modified in a long time: we probably want to report the `date of the last "scan" of the bucket`, i.e. (when trailing oplog) something like the date of the last metadata operation which has been computed in the utilization metrics.

Reply:
Yes I meant the current cseq (not for a specific bucket: if the bucket is not updated, the "date of the last scan of the bucket" will never happen, unless we repair). I'll update the ticket, the comment remains fine to me.

Reply:
i think the comment may be misleading for someone discovering the issue here: if we update the ticket, we might as well update the comment to avoid any confusion...